src/run.rs

```diff
@@ -678,14 +679,16 @@ fn vacuum(
         );
 
         // Start deleting images, beginning with the least recently used.
-        for (image_id, _) in sorted_image_nodes {
-            // Delete the image.
-            if let Err(error) = delete_image(image_id) {
-                // The deletion failed. Just log the error and proceed.
-                error!("{}", error);
-            } else {
-                // Forget about the deleted image.
-                deleted_image_ids.insert(image_id.clone());
+        for image_ids in sorted_image_nodes.chunks(usize::try_from(del_chunk_size).unwrap()) {
```

Comment:
The `unwrap` here makes me feel uneasy. We can avoid it by just making the chunk size unsigned in the first place, right?

Reply:
updated